css/legacy/includes/_styles.scss

```diff
@@ -702,6 +702,7 @@
       bottom: 0;
       left: 0;
       right: 0;
+      white-space: initial;
```

Comment:
To have a more global fix, maybe this can be applied to `.rich_text_container` instead.